hack/install-kustomize.sh

```diff
@@ -9,6 +9,7 @@ arch=$(go env GOARCH)
 source_url="https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2F${required_version}/kustomize_${required_version}_${os}_${arch}.tar.gz"
 target_dir="${script_dir}/../bin"
 target_path="${target_dir}/kustomize"
+target_archive="${target_dir}/kustomize.tar.gz"
```

Comment:
Makes sense, downloading a file to target_path and extracting the tar to the same path does not make any sense. I hope we don't have this issue in other scripts.

Reply:
I haven't seen it yet. Will check.